packages/web-components/fast-components/src/design-system-provider/README.md

```diff
@@ -0,0 +1,251 @@
+# Design System Provider
```

Comment:
a small table of contents at the start may be helpful for traversing this document.